src/server.js

```diff
@@ -0,0 +1,17 @@
+const http = require('http');
+const url =require('url');
+const {airportInformation} = require('./airportDetailsProvider.js');
+const host = 'localhost';
+const port = 3000;
```

Comment:
-> .env
Also add .env.template file, which contains only keys of all env variables used. Ok to push this file.